src/shell/Models/DevWallet.cs

```diff
@@ -7,110 +7,113 @@
 
 namespace NeoShell.Models
 {
-  class DevWallet : Wallet
-  {
-    private readonly Dictionary<UInt160, DevWalletAccount> accounts = new Dictionary<UInt160, DevWalletAccount>();
-
-    public override string Name { get; }
-    public override Version? Version => null;
-
-    public DevWallet(ProtocolSettings settings, string name, IEnumerable<DevWalletAccount>? accounts = null) : base(string.Empty, settings)
+    class DevWallet : Wallet
```

Comment:
BCTK Lib has a [ToolkitWallet](https://github.com/ngdenterprise/neo-blockchaintoolkit-library/blob/master/src/bctklib/models/ToolkitWallet.cs) class you could use instead of DevWallet. I plan to move express over to ToolkitWallet at some point in the future

Reply:
I'm OK with doing this as part of a general cleanup pass. Please open an issue to track and include any other areas of the code you think need to be addressed. I have thoughts as well, but I would like to hear your thoughts on eliminating technical debt here.